src/components/forms/Select/Select.tsx

```diff
@@ -171,10 +172,10 @@ const Select: React.FC<SelectProps> = ({
           await setTouched(true);
           await validateField(name);
         }}
-        aria-describedby={errorMessageId}
+        aria-describedby={[descriptionId, errorMessageId].filter(Boolean).join(' ')}
```

Comment:
this makes the description linking to the input more inconsistent, since anything that's `TextField` based does *not* have this behaviour.

Please revert this and keep it out of scope for this ticket, and solve it for *all* the components in one go in a separate ticket.

I do understand the intent, but I feel that inconsistency is worse than not having this at this point (the legacy renderer also does not have this and it has never come up in a11y audits)